site/reference/fn/eval/README.md

```diff
@@ -62,24 +62,17 @@ kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/ex
 ```sh
 # read the Resources from DIR, provide them to a container my-fun as input,
 # write my-fn output back to DIR
-kpt fn run DIR/ --image gcr.io/example.com/my-fn
+kpt fn eval DIR/ --image gcr.io/example.com/my-fn
```

Comment:
Where is it documented that an image or exec must be provided. Where's the check and test for it?

Reply:
A check has added. The documentation will be updated later.